options.go

```diff
@@ -395,15 +397,40 @@ func (o *Options) CLI() serpent.OptionSet {
 				"The envbuilder subsystem is always included.",
 		},
 	}
+
+	// Add options without the prefix for backward compatibility. These options
+	// are marked as deprecated and will be removed in future versions. Note:
+	// Future versions will require the 'ENVBUILDER' prefix for default
+	// environment variables.
+	for _, o := range set {
+		if strings.HasPrefix(o.Env, prefix) {
+			prevOption := o
+			prevOption.Env = strings.TrimPrefix(o.Env, prefix)
+			prevOption.UseInstead = []serpent.Option{o}
+			prevOption.Hidden = true
+			set = append(set, prevOption)
+		}
+	}
+
+	return set
 }
 
 func (o *Options) Markdown() string {
-	cliOptions := o.CLI()
+	var visibleOpts serpent.OptionSet
+
+	// We don't want to include deprecated opts in the README.
+	for _, opt := range o.CLI() {
```

Comment:
Technically speaking, you can move this for-range loop to another function and have:

`cliOptions := skipDeprecatedOptions(cliOptions)`